heareval/tasks/nsynth_pitch.py

```diff
@@ -97,7 +97,7 @@ def get_split_metadata(self, split: str) -> pd.DataFrame:
             )
         )
 
-        return metadata[["relpath", "slug", "subsample_key", "split", "label"]]
```

Comment:
This line fixes the Nsynth Pitch Bug. It returns the data frame with all the columns. #157